components/icons/IconGlobe.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Icon, IconProps } from '@chakra-ui/react';
+
+export function IconGlobe(props: IconProps) {
+    return (
+        <Icon viewBox="0 0 24 24" width="24px" height="24px" {...props}>
```

Comment:
As a wider discussion, how should we be approaching icons. Icon pack or just add them as needed?

Reply:
We just add them as needed.